### DISCUSSION TITLE
skynet开源交流社区qq群，欢迎热爱技术的朋友加入交流

### DISCUSSION THREAD

**Discussion by G0561** [cloudwu/skynet#1672] [General]
大家好！为方便大家及时交流skynet技术心得！有需要的朋友可以加下面群

skynet开源社区 群号：815491157
skynet开源社区 群号：807474118

**Comment by wangzhione** [comment]
有微信的吗, 有些人没有qq

**Comment by sniper00** [comment]
> 有微信的吗, 有些人没有qq

云大不用微信
